### ISSUE TITLE
parmed.py: several errors with command lines

### ISSUE THREAD

**Issue by hainm** [ParmEd/ParmEd#176]
hey man,

I got several errors and want to report here

* Using `ls`,  got 

```
Loaded Amber topology file C2Me_GC.prmtop
Reading input from STDIN...
> 
> ls
Traceback (most recent call last):
  File "/home/haichit/anaconda3/envs/parmed_test/bin/parmed.py", line 23, in <module>
    clapp()
  File "/home/haichit/anaconda3/envs/parmed_test/lib/python3.4/site-packages/parmed/scripts.py", line 189, in clapp
    parmed_commands.cmdloop()
  File "/home/haichit/anaconda3/envs/parmed_test/lib/python3.4/cmd.py", line 138, in cmdloop
    stop = self.onecmd(line)
  File "/home/haichit/anaconda3/envs/parmed_test/lib/python3.4/cmd.py", line 217, in onecmd
    return func(arg)
  File "<string>", line 1, in <lambda>
  File "/home/haichit/anaconda3/envs/parmed_test/lib/python3.4/site-packages/parmed/tools/parmed_cmd.py", line 152, in _normaldo
    self.stdout.write('%s\n' % action)
TypeError: __str__ returned non-string (type bytes)
```

(oops, will update more if I get more error).

**Comment by hainm** [comment]
fix: https://github.com/hainm/ParmEd/commit/97991413dfb83e7d5e1c29d97ea8cc1f5915c63f

**Comment by swails** [comment]
Fixed by #177
